Assets/Scripts/UI/InGameUI/MenuLeft.cs

```diff
@@ -33,6 +33,9 @@ public void OpenMenu(string menuName)
 
         menu.SetActive(true);
         CurrentlyOpen = menu;
+
+		WorldController.Instance.soundController.PlayClick();
```

Comment:
I don't think you are using stylecop since your code deviates from the project standards. you can [follow this wiki article to install it here](https://github.com/TeamPorcupine/ProjectPorcupine/blob/master/CONTRIBUTING.md#style-guidelines)